src/components/Search/SearchPageHeader.tsx

```diff
@@ -33,59 +32,50 @@ import type {SearchDataTypes, SearchReport} from '@src/types/onyx/SearchResults'
 import type DeepValueOf from '@src/types/utils/DeepValueOf';
 import type IconAsset from '@src/types/utils/IconAsset';
 import {useSearchContext} from './SearchContext';
+import SearchRouterInput from './SearchRouter/SearchRouterInput';
 import type {SearchQueryJSON} from './types';
 
-type HeaderWrapperProps = Pick<HeaderWithBackButtonProps, 'title' | 'subtitle' | 'icon' | 'children'> & {
-    subtitleStyles?: StyleProp<TextStyle>;
+type HeaderWrapperProps = Pick<HeaderWithBackButtonProps, 'icon' | 'children'> & {
+    content: string;
```

Comment:
I'd rename this something more simple like `text` or `message` or similar. 
Content suggests to me that this is a more complex structure, and at first I was confused why we have both `children` and `content`